packages/kbn-securitysolution-io-ts-list-types/src/common/pit/index.ts

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import * as t from 'io-ts';
+
```

Comment:
Can we use `SavedObjectsPitParams` from core? Or is it we can't import it into the package here?

Reply:
You cannot import them here from `core`. They might not make sense to your REST API validation if you're wanting to keep the types the same.

Reply:
I see, I see. I missed thinking about the way you're using it as validations as well here 👍🏽